src/app/ai-knowledge-base/pages/ai-knowledge-base-search/dialogs/aiknowledge-base-create-update/aiknowledge-base-create-update.component.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing'
+import { ComponentFixture, TestBed } from '@angular/core/testing'
+import { FormsModule, ReactiveFormsModule } from '@angular/forms'
+import { ActivatedRoute } from '@angular/router'
+import { LetDirective } from '@ngrx/component'
+import { BreadcrumbService, PortalCoreModule } from '@onecx/portal-integration-angular'
+import { TranslateTestingModule } from 'ngx-translate-testing'
+import { AIKnowledgeBaseCreateUpdateComponent } from './aiknowledge-base-create-update.component'
+
+Object.defineProperty(window, 'matchMedia', {
+  writable: true,
+  value: jest.fn().mockImplementation((query) => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // Deprecated
+    removeListener: jest.fn(), // Deprecated
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn()
+  }))
+})
+
+describe('AIKnowledgeBaseCreateUpdateComponent', () => {
+  let component: AIKnowledgeBaseCreateUpdateComponent
+  let fixture: ComponentFixture<AIKnowledgeBaseCreateUpdateComponent>
+
+  const mockActivatedRoute = {}
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [AIKnowledgeBaseCreateUpdateComponent],
+      imports: [
+        PortalCoreModule,
+        FormsModule,
+        ReactiveFormsModule,
+        LetDirective,
+        TranslateTestingModule.withTranslations(
+          'en',
+          // eslint-disable-next-line @typescript-eslint/no-require-imports
```

Comment:
Do you need to disable eslint ? In src/app/ai-knowledge-base/pages/ai-knowledge-base-details/ai-knowledge-base-details.component.spec.ts you didn't use this